CalendarApp/Models/RemoteChange.swift

```diff
@@ -7,40 +7,16 @@
 
 import Foundation
 
-@objc public enum ChangeType: Int32 {
-    case Delete = 0
-    case Create = 1
-    case Update = 2
-}
-
 @objcMembers
-public class RemoteChange : NSObject {
+public class RemoteChange : NSObject, Revision {
```

Comment:
Why not have the initializer make the fields? It's odd to see some of these defaults. Like, would NoChange ever be valid?